hooks/tk-multi-publish2/nuke/publish_comp.py

```diff
@@ -165,8 +84,7 @@ def validate(self, task_settings, item):
         if item.properties.get("node"):
             status = self._bbsize(item) and status
             if item.properties['fields'].get('output') == 'main':
-                status = self._sync_frame_range(item) and status
```

Comment:
Can we add back this? Because `_sync_frame_range()` is only called in super for the nuke session, and not the node.
I know in general people publish session and nodes at one go, but from a strictly validation point of view, considering main node as an independent item, we want to verify nuke is in sync with shotgun and the rendered range is in sync with nuke.

Reply:
Done